app/src/test/java/org/mozilla/fenix/crashes/CrashReporterControllerTest.kt

```diff
@@ -16,64 +16,65 @@ import org.junit.Test
 import org.mozilla.fenix.R
 import org.mozilla.fenix.components.Components
 import org.mozilla.fenix.components.metrics.Event
+
 import org.mozilla.fenix.utils.Settings
 
 class CrashReporterControllerTest {
```

Comment:
This test fails: I think you need to add the NavGraphTestRule.

While we're modifying this file, I think we should avoid the churn of renaming the `navController`: this is already a big change. Can you change it back?